pom.xml

```diff
@@ -374,6 +426,17 @@
           <check/>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.sonatype.plugins</groupId>
+        <artifactId>nexus-staging-maven-plugin</artifactId>
+        <version>1.6.7</version>
```

Comment:
Minor thing but can we use the latest version [1.6.8](https://search.maven.org/artifact/org.sonatype.plugins/nexus-staging-maven-plugin/1.6.8/maven-plugin)?